sdks/java/io/parquet/src/main/java/org/apache/beam/sdk/io/parquet/ParquetIO.java

```diff
@@ -165,6 +171,10 @@ public Read from(String filepattern) {
       return from(ValueProvider.StaticValueProvider.of(filepattern));
     }
 
+    public Read withAvroDataModel(GenericData model) {
```

Comment:
Could you add Javadoc for this public method and maybe an example into main class?

Reply:
Will do.